database.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+from os import path, makedirs
+from datetime import datetime
+from sqlalchemy import DateTime, ForeignKey, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship, mapped_column
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+
+Base = declarative_base()
+DB_PATH = path.join("db", "database.db")
+
+# Create the folder if it doesn't exist
+folder_path = path.dirname(DB_PATH)
+if not path.exists(folder_path):
+    makedirs(folder_path)
+
+
+class User(Base):
+    """Class representing a user in the system."""
+
+    __tablename__ = 'users'
+
+    id = mapped_column(Integer, primary_key=True)
+    email = mapped_column(String(255), unique=True, nullable=True)
+
+    uploads = relationship("Upload", back_populates="user", cascade="all, delete-orphan")
+
+
+class Upload(Base):

```

Comment:
I would add even more properties and functions, like processing_time, get for output as well..